verification/curator-service/ui/cypress/plugins/index.ts

```diff
@@ -0,0 +1,19 @@
+// Need to use require in plugins file 
+// https://stackoverflow.com/questions/59743577/is-there-anyway-to-query-mongodb-in-cypress-test
+const MongoClient = require('mongodb').MongoClient;
+const url = "mongodb://localhost:27017/";
```

Comment:
is there any way to reuse the connection data defined in the .env file? this could get out of sync

Reply:
It seems the port is hardcoded here: https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/dev/docker-compose.yml

But good news is that if the port becomes out of sync the test fails with a clear error message (couldn't connect to local mongo). I will add that file to the integration tests yml change detection so if that file changes then the integration tests will run on CI.